lib/ci-helper.ts

```diff
@@ -420,6 +424,7 @@ export class CIHelper {
 
         if (optionsUpdated && updateOptionsInRef) {
             await this.notes.set("", options, true);
+            await this.notes.push(this.urlRepo);
```

Comment:
Is `updateOptionsInRef` still useful?  It seems outdated and would allow the update if needed which would simplify upstream callers.  `optionsUpdated` does not need to return true if the notes are actually updated but that only matters if callers use it.

Reply:
It is totally outdated, thank you for pointing that out! I will drop it in a new commit.